src/utils/exceptions.ts

```diff
@@ -23,6 +23,11 @@ export class GenericUserErrorException extends Error {
     super(message);
   }
 }
+export class GenericGameErrorException extends Error {
+  constructor(message = "User mistake in game endpoint") {
+    super(message);
+  }
+}
```

Comment:
Make an abstract class for a game exception and let all the game exceptions extend it. This will make error handling easier